### ISSUE TITLE
CAMPD tool tips- 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2446]
## Context 

Testing for 1910 complete. 1 potential issue. For the preview data tooltip, there can be a case where the user navigates to the tooltip and the preview data button is disabled. For screenreaders, disabled elements are not announced. May need to add to the text something like 'Preview button will be disabled until at least one filter is selected'

Note: Verify with Anthony if the data preview tool tip should update dynamically.

**Comment by aprematta** [comment]
testing complete, tooltip text is dependent on the state and appropriate
